packages/cactus-plugin-ledger-connector-corda/src/main/typescript/web-services/deploy-contract-jars-endpoint.ts

```diff
@@ -140,75 +142,12 @@ export class DeployContractJarsEndpoint implements IWebServiceEndpoint {
     }
   }
 
-  private async doDeploy(
```

Comment:
@petermetz I changed this code for a call to the container endpoint, do you prefer me to revert to the previous function?

Reply:
@elenaizaguirre No, it's the right call to keep things in the JVM implementation as long as that works.